.github/workflows/build-macos-installers.yml

```diff
@@ -250,6 +237,88 @@ jobs:
           name: chia-installers-macos-dmg-${{ matrix.os.name }}
           path: ${{ github.workspace }}/build_scripts/final_installer/
 
+      - name: Remove working files to exclude from cache
+        run: |
+          rm -rf ./chia-blockchain-gui/packages/gui/daemon
+
+  publish:
+    name: Publish ${{ matrix.os.name }} DMG
+    runs-on: ${{ matrix.os.runs-on }}
+    needs:
+    - build
+    timeout-minutes: 5
+    strategy:
+      fail-fast: false
+      max-parallel: 4
```

Comment:
```suggestion
```
Same comment as before.. We should probably remove the other max-parallel in this file as well.